BUILD.md

```diff
@@ -295,7 +295,7 @@ $ ./sbt namerd/dcos:docker
 $ docker run -p 2181:2181 -p 4180:4180 -v /path/to/repo:/myapp -w /myapp buoyantio/namerd:0.9.1-SNAPSHOT-dcos namerd/examples/zk.yaml
 ```
 
-[funsuite]: http://www.scalatest.org/getting_started_with_fun_suite
+[funsuite]: https://www.scalatest.org/getting_started_with_fun_suite
 [l5d-ci]: https://circleci.com/gh/linkerd/linkerd
 [sbt]: https://www.scala-sbt.org/
-[scalatest]: http://www.scalatest.org/
+[scalatest]: https://www.scalatest.org/
```

Comment:
Thanks a bunch for your contribution! Unfortunatelly the TLS of scalatest.org seems to be having some problems for me and I get: 

<img width="877" alt="Screenshot 2020-01-13 at 13 36 49" src="https://user-images.githubusercontent.com/4391506/72253306-ebfc2f80-3609-11ea-876e-5b9afa4fd21b.png">

Therefore I am not entirely convinced moving these to https is a win